### ISSUE TITLE
Linter breaks on embedded SVG

### ISSUE THREAD

**Issue by mbarlock** [htmllint/htmllint#94]
When encountering an SVG with any svg tags the parser gives an error similar to this

```
Fatal error: Index passed to line/column function (27778) does not keep with order (last was 27790)
```

The issue seems to be that the list in is_void_elements is incomplete

**Comment by chris-bmj** [comment]
Yup I'm experiencing this too. Annoying for me as I'm using htmllint in a script and the exception causes problems.

**Comment by goldcaddy77** [comment]
For anybody else that encounters this issue, this can also simply be caused by an unclosed bracket on a tag:

```html
<my-custom-tag
  custom-param-1="hello world"
</my-custom-tag>
```

Notice the missing `>` to close the tag.

**Comment by jitendravyas** [comment]
@goldcaddy77 ok but how to find that instance?

**Comment by goldcaddy77** [comment]
Not sure how I found it - maybe syntax highlighting?  You might be able to run it through an xhtml validator?

**Comment by mlochbaum** [comment]
The non-closed tag problem appears to have been fixed by commit 312b469. Maybe the svg issue was fixed too? In any case it didn't break when I gave it an svg.

Linting is still very broken for svgs; htmllint will need some sort of mode system to deal with these.
